pallets/xst/src/lib.rs

```diff
@@ -292,7 +368,7 @@ pub mod pallet {
         /// Asset that is used to compare collateral assets by value, e.g., DAI.
         pub reference_asset_id: T::AssetId,
         /// List of tokens enabled as collaterals initially.
-        pub initial_synthetic_assets: Vec<T::AssetId>,
+        pub initial_synthetic_assets: Vec<(AssetSymbol, AssetName, T::Symbol, Fixed)>,
```

Comment:
I think it's better to pass AssetId here and register assets via `assets::GenesisConfig`